### ISSUE TITLE
QA_EXPT-9 & EXPT-10_Procedure_content-is-same

### ISSUE THREAD

**Issue by BSravanthi** [Virtual-Labs/analog-cmos-vlsi-circuit-design-iiith#32]
Defect Description :
In the Procedure page of 9th and 10th experiments of this lab, found that the content is same.

Actual Result :
In the Procedure page of 9th and 10th experiments of this lab, found that the content is same.

Environment :
OS: Windows 7, Ubuntu-16.04,Centos-6
Browsers:Firefox-42.0,Chrome-47.0,chromium-45.0
Bandwidth : 100Mbps
Hardware Configuration:8GBRAM ,
Processor:i5

Attachments:
![qa_oe_avlsi_i29](https://cloud.githubusercontent.com/assets/13479177/26664636/a65b317e-46b0-11e7-9539-fee0c3a93e11.png)

**Comment by BSravanthi** [comment]
Referring to #27

**Comment by sravanthimodepu** [comment]
Yes.. The 9th & 10th experiments belongs to the same content but the simulations are different. Found the same from the vlab.co.in page.

**Comment by BSravanthi** [comment]
Need to contact lab developer. The issue will be opened till we get the response.
